addons/account/views/account_move_views.xml

```diff
@@ -1075,7 +1075,7 @@
                                                domain="[('partner_id', '=', bank_partner_id)]"
                                                attrs="{'readonly': [('state', '!=', 'draft')]}"/>
                                         <field name="qr_code_method"
-                                               attrs="{'invisible': [('display_qr_code', '=', False)]}"/>
+                                               attrs="{'invisible': ['|', ('display_qr_code', '=', False), ('move_type', '!=', 'out_refund')]}"/>
```

Comment:
In the task description you say that a QR code for payment on a vendor bill makes no sense.
I do agree with this, but then... Why are you making the `qr_code_method` **visible only for credit notes**?

Shouldn't it be like... ?
```py
attrs="{'invisible': ['|', ('display_qr_code', '=', False), ('move_type', '=', 'out_refund')]}"/>
```
Even if you select the Swiss QR Bill, the resulting Credit Note won't have the QR code anyway.
runbot: https://20656226-14-0-all.runbot124.odoo.com/web#id=19&action=393&model=account.move&view_type=form&cids=3&menu_id=277

Reply:
@lordkrandel totally agree, my bad miss-typing ! thank you

Reply:
👍🏻 
As a constructive feedback, please care that this shows that you did not re-test it after modification.
(I'm also always overlooking this, so I can relate)